src/System.IO.Pipelines/StreamExtensions.cs

```diff
@@ -29,21 +29,7 @@ public static IPipeWriter AsPipelineWriter(this Stream stream)
         public static IPipeWriter AsPipelineWriter(this Stream stream, IBufferPool pool)
         {
             var pipe = new Pipe(pool);
-            pipe.CopyToAsync(stream).ContinueWith((task, state) =>
-            {
-                var innerPipe = (Pipe)state;
-
-                if (task.IsFaulted)
-                {
-                    innerPipe.Reader.Complete(task.Exception.InnerException);
-                }
-                else
-                {
-                    innerPipe.Reader.Complete();
-                }
-            }, 
-            pipe, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default);
-
+            pipe.CopyToAsync(stream);
```

Comment:
Hahaha, yeah sorry I wasn't really thinking about the Pipe -> Stream as it wasn't my original problem but you are right the whole thing needs to be consistent and those other API's could be removed. I was trying for a surgical strike rather than open heart surgery... give me a few minutes to fix it up